### ISSUE TITLE
Get Single Author

### ISSUE THREAD

**Issue by MaggieChafee** [MaggieChafee/CapstoneServerSide#17]
# User Story

- Given a user selects an author’s name from either the single book details page or the view all authors page
- Then the user should be directed to the single author’s page where they can see the author's information and books. 

# Acceptance Criteria
When tested, this endpoint should return the following information:
- author's information
- books the author has written or co-written

# Dependencies
This is a Stretch goal. All other MVP functionality should be completed first. 

# Dev Notes

This is a GET request for a single entity. Here is a similar example:
```csharp
app.MapGet("/orders/{id}", (HhpwDbContext db, int id) =>
            {
                var singleOrder = db.Orders.SingleOrDefault(o => o.Id == id);
                if (singleOrder == null) 
                {
                    return Results.NotFound();
                }
                return Results.Ok(singleOrder);
            });
```
